crates/passes/src/transform/insert_call.rs

```diff
@@ -13,7 +13,7 @@ impl Transform for InsertCallNodes {
             let node = graph.get_node(idx);
             match FuncNode::concrete(node) {
```

Comment:
Using an `if let` construct might make this code cleaner as you don't do anything on `None` case